manim/mobject/geometry/labeled_shapes.py

```diff
@@ -0,0 +1,151 @@
+r"""Mobjects that inherit from lines and contain a label along the length."""
+
+from __future__ import annotations
+
+__all__ = ["LabeledLine", "LabeledArrow"]
+
+from manim.constants import *
+from manim.mobject.geometry.line import Arrow, Line
+from manim.mobject.geometry.shape_matchers import (
+    BackgroundRectangle,
+    SurroundingRectangle,
+)
+from manim.utils.color import *
+
```

Comment:
This is for type checking stuff in the constructor params list
```suggestion
from manim.mobject.text.tex_mobject import Tex, MathTex
from manim.mobject.text.text_mobject import Text
from manim.utils.color import Color
```

Reply:
Thanks, I'll add this suggestion.